tests/integration/goldens/asset/samples/generated_samples/snippet_metadata_v1_asset_v1.json

```diff
@@ -0,0 +1,1048 @@
+{
+  "snippets": [
+    {
+      "clientMethod": {
+        "async": true,
+        "method": {
+          "fullName": "AnalyzeIamPolicyLongrunning",
+          "service": {
+            "shortName": "AssetService"
+          }
+        }
+      },
+      "file": "samples/generated_samples/cloudasset_generated_asset_v1_asset_service_analyze_iam_policy_longrunning_async.py",
```

Comment:
This path should be relative to the location of this metadata file. Just double checking that's the case.

Reply:
Ah it's currently relative to the library root.  I'll remove the `samples/generated_samples` since the metadata sits in the same directory as the snippets.